pkgs/by-name/ha/handheld-daemon/package.nix

```diff
@@ -0,0 +1,54 @@
+{ config
+, fetchFromGitHub
+, hidapi
+, kmod
+, lib
+, python3
+, toybox
+}:
+python3.pkgs.buildPythonApplication rec {
+  pname = "handheld-daemon";
+  version = "1.1.0";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "hhd-dev";
+    repo = "hhd";
+    rev = "abe34c6841476f5b41afe30ee18ff3e510402d68";
```

Comment:
I don't know if it is helpful context, but hhd is versioned and also provided as a source tar ball from pypi.

https://pypi.org/project/hhd/

This nix pr builds from GitHub, but copr and aur both build from pypi.

So it may be better without the v

Reply:
In general I prefer builds from GitHub; that way it's more obvious that the source is what you're building. Pypi tarballs are provided by the uploader, so can be quite different from the source on GitHub...

Reply:
It's better to use `rev = "v${version}" (or rev = "refs/tags/v${version}" for the more more prudent-minded) to facilitate package updates.